presto-docs/src/main/sphinx/release/release-339.rst

```diff
@@ -0,0 +1,70 @@
+=========================
+Release 339 (XX Jul 2020)
+=========================
+
+General Changes
+---------------
+
+* Add :func:`approx_most_frequent`. (:issue:`3425`)
+* Physical bytes scan limit for queries can be configured via ``query.max-scan-physical-bytes`` configuration property
+  and ``query_max_scan_physical_bytes`` session property. (:issue:`4075`)
+* Remove support for addition and subtraction between ``TIME`` and ``INTERVAL YEAR TO MONTH`` types. (:issue:`4308`)
+* Fix planning failure when join criteria contains subqueries. (:issue:`4380`)
+* Fix failure when subqueries appear in window function arguments. (:issue:`4127`)
+* Fix failure when subquery in WITH clause contains hidden columns. (:issue:`4423`)
+* Fix failure when referring to type names with different case ``GROUP BY`` queries. (:issue:`2960`)
+* Fix failure for queries involving ``DISTINCT`` when expressions in ``ORDER BY`` clause differ from expressions in ``SELECT`` clause by case. (:issue:`4233`)
+* Fix incorrect type reporting for ``TIMESTAMP`` and ``TIMESTAMP WITH TIME ZONE`` for legacy clients. (:issue:`4408`)
+* Fix failure when querying nested ``TIMESTAMP`` or ``TIMESTAMP WITH TIME ZONE`` for legacy clients. (:issue:`4475`, :issue:`4425`)
+
+JDBC Driver Changes
+-------------------
+
+* Fix reading ``timestamp`` and ``timestamp with time zone`` values with year number higher than 9999
+  or negative. (:issue:`4364`
+
+Druid Connector Changes
+-----------------------
+
+* Fix handling of table and column names containing non-ASCII characters. (:issue:`4312`)
+
+Hive Connector Changes
+----------------------
+
+* Make ``location`` parameter of the ``system.register_partition`` procedure optional. (:issue:`4443`)
```

Comment:
Update docs to reflect that (see hive.rst)

Reply:
That's out of scope for this PR. Once the documentation is added, we can link to it here.